e2e/cypress/integration/specs/system/cookie-consent.b2c.e2e-spec.ts

```diff
@@ -0,0 +1,41 @@
+import { at } from '../../framework';
+import { HomePage } from '../../pages/home.page';
+
+describe('Cookie Consent', () => {
+  describe('starting at home page', () => {
+    before(() => {
+      HomePage.navigateTo();
+    });
+
+    it('should see cookie banner', () => {
+      at(HomePage, () => {
+        cy.get('.cookies-banner').should('contain', 'Your privacy is important for us');
+      });
+    });
+
+    it('should accept all cookies', () => {
+      at(HomePage, () => {
+        cy.getCookies().then(cookies => {
+          expect(cookies[cookies.length - 1]).not.to.have.property('name', 'cookieConsent');
+        });
+
+        cy.get('.cookies-banner .btn.btn-light').click();
+        cy.wait(3000);
```

Comment:
`waitLoadingEnd` does not really fit here since it is not related to a loading div but a complete page reload. For this the `waitLoadingEnd`500ms are to short. Kept `cy.wait(3000);`.

Reply:
`waitLoadingEnd` can also be supplied with a longer initial wait. But if the page is reloaded it shouldn't be necessary. 😉